### PR TITLE
update Node.js opbeans docker file for v2 API

### PR DESCRIPTION
**Pull request by Qard**
Depends on https://github.com/elastic/opbeans-node/pull/36

### CODE REVIEW

File: docker/opbeans/node/Dockerfile [hunk 1]
```diff
@@ -2,10 +2,6 @@ FROM opbeans/opbeans-node:latest
 
 ENV NODE_ENV=production
 
-RUN npm install pm2 -g
-
 COPY entrypoint.sh /app/entrypoint.sh
-COPY processes.config.js /app/processes.config.js
 
-CMD ["pm2-runtime", "processes.config.js"]
```
Comment (watson) [inline docker/opbeans/node/Dockerfile]: As per [comment in opbeans-node](https://github.com/elastic/opbeans-node/pull/36#discussion_r219741691), I suggest we leave this line in but change it to use a `ecosystem-workload.config.js` config file.

Comment (Qard) [reply docker/opbeans/node/Dockerfile]: fixed.

**Comment by watson** [comment]
I completely forgot that this was already opened when I made #210. Sorry 😅

**Comment by Qard** [comment]
No worries. :)

**Pull request closed without merging**